pkg/energyefficientscheduling/energyefficientscheduling.go

```diff
@@ -1,37 +1,142 @@
 package energyefficientscheduling
 
 import (
+	"fmt"
+	"math"
 	"context"
 
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	klog "k8s.io/klog/v2"
 	"k8s.io/kubernetes/pkg/scheduler/framework"
+	"github.com/paypal/load-watcher/pkg/watcher"
+
+	pluginConfig "sigs.k8s.io/scheduler-plugins/apis/config"
+	"sigs.k8s.io/scheduler-plugins/apis/config/v1beta2"
 )
 
-const Name = "Peaks"
+cconst (
```

Comment:
The typo `cconst` should be `const`. 

We need a git action for syntax and format check to avoid typos.

Reply:
Hi, Thankyou for the feedback. can you review this PR https://github.com/sustainable-computing-io/scheduler-plugins/pull/1